api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/rules/validationrules/impl/SchoolAgedIndigenousSupportRule.java

```diff
@@ -0,0 +1,63 @@
+import ca.bc.gov.educ.studentdatacollection.api.constants.StudentValidationFieldCode;
+import ca.bc.gov.educ.studentdatacollection.api.constants.StudentValidationIssueSeverityCode;
+import ca.bc.gov.educ.studentdatacollection.api.constants.StudentValidationIssueTypeCode;
+import ca.bc.gov.educ.studentdatacollection.api.rules.ValidationBaseRule;
+import ca.bc.gov.educ.studentdatacollection.api.struct.StudentRuleData;
+import ca.bc.gov.educ.studentdatacollection.api.struct.v1.SdcSchoolCollectionStudentValidationIssue;
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *  | ID  | Severity | Rule                                                                         | Dependent On |
+ *  |-----|----------|------------------------------------------------------------------------------|--------------|
+ *  | V77 | WARNING  |  Only school-aged students should be reported with Indigenous Support        |  V04, V30    |
+ *                      Programs.
+ *
+ */
+@Component
+@Slf4j
+@Order(680)
+public class SchoolAgedIndigenousSupportRule implements ValidationBaseRule {
+
+    @Override
+    public boolean shouldExecute(StudentRuleData studentRuleData, List<SdcSchoolCollectionStudentValidationIssue> validationErrorsMap) {
+        log.debug("In shouldExecute of AdultIndigenousFundingRule-V77: for collectionType {} and sdcSchoolCollectionStudentID :: {}" , FteCalculatorUtils.getCollectionTypeCode(studentRuleData),
+                studentRuleData.getSdcSchoolCollectionStudentEntity().getSdcSchoolCollectionStudentID());
+
+        var shouldExecute = StringUtils.isNotEmpty(studentRuleData.getSdcSchoolCollectionStudentEntity().getDob()) &&
+                StringUtils.isNotEmpty(studentRuleData.getSdcSchoolCollectionStudentEntity().getEnrolledProgramCodes()) &&
+                isValidationDependencyResolved("V77", validationErrorsMap);
+
+        log.debug("In shouldExecute of AdultIndigenousFundingRule-V77: Condition returned  - {} for sdcSchoolCollectionStudentID :: {}" ,
+                shouldExecute,
+                studentRuleData.getSdcSchoolCollectionStudentEntity().getSdcSchoolCollectionStudentID());
+
+        return shouldExecute;
+    }
+
+    @Override
+    public List<SdcSchoolCollectionStudentValidationIssue> executeValidation(StudentRuleData studentRuleData) {
+        log.debug("In executeValidation of AdultIndigenousFundingRule-V77 for sdcSchoolCollectionStudentID ::" + studentRuleData.getSdcSchoolCollectionStudentEntity().getSdcSchoolCollectionStudentID());
+
+        final List<SdcSchoolCollectionStudentValidationIssue> errors = new ArrayList<>();
+        var student = studentRuleData.getSdcSchoolCollectionStudentEntity();
+
+        if (BooleanUtils.isFalse(student.getIsSchoolAged())) {
```

Comment:
Indigenous program check is missing. 
student.getIsSchoolAged() will be set after the rules have executed with no errors. This here will lead to null pointer. DOBUtil has functions that you can use to determine school-aged and adult flags